modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -1769,7 +1771,7 @@ private CompletableFuture<?> processMultiEntryAction(ReadWriteMultiRowReplicaReq
                 }
 
                 return allOf(pkReadLockFuts).thenCompose(ignore -> {
-                    Collection<BinaryRow> result = new ArrayList<>();
+                    Collection<Boolean> result = new ArrayList<>();
```

Comment:
`java.util.BitSet` will be more effective.

Reply:
This approach is not mine. If you think it really needs to change the return type, please create another issue.